pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java

```diff
@@ -641,4 +643,131 @@ public CompletableFuture<Boolean> validateTenantAdminAccess(String tenantName, S
                 });
     }
 
+    @Override
+    public CompletableFuture<Void> removePermissionsAsync(TopicName topicName) {
```

Comment:
Could you please add an NPE check for `topicName` since it is a public API?

Reply:
Maybe a bit redundant